.github/workflows/build_docs.yml

```
@@ -0,0 +1,36 @@
+name: build_docs
+
+on: [push]
```

So muesste das dann, wenn er nur beim Merge/Push in den Master laufen soll

```suggestion
on:
  push:
    branches:
      - master
```

---

Ist eingebracht - Danke